MovieApp/MovieApp/ViewController/ViewController.swift

```diff
@@ -112,6 +116,10 @@ class ViewController: UIViewController {
         present(vc, animated: true, completion: nil)
     }
     
+    @IBAction func didTapMenu(_ sender: Any) {
```

Comment:
See if you can delete this. Also check your UICollectionView Outlets, they have warnings in the Storyboard